patches/minecraft/net/minecraft/client/renderer/entity/RenderZombieVillager.java.patch

```diff
@@ -0,0 +1,10 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/entity/RenderZombieVillager.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/entity/RenderZombieVillager.java
+@@ -25,6 +25,7 @@
+ 
+     protected ResourceLocation func_110775_a(EntityZombieVillager p_110775_1_)
+     {
++        if (true) return p_110775_1_.getForgeProfession().getZombieSkin();
+         switch (p_110775_1_.func_190736_dl())
```

Comment:
This is actually legit, seems i skipped/forgot about this when i was doing it... could of sworn i did skins tho...